Library/List/RefundList.cs

```diff
@@ -7,8 +8,9 @@ namespace Recurly
     internal class RefundList : IEnumerable<Refund>
     {
         private List<Refund> Refunds = new List<Refund>();
-        private Invoice.RefundMethod RefundMethod;
+        private Invoice.RefundOptions RefundOptions;
 
+        [Obsolete("This constructor is deprecated, please use RefundList(IEnumerable<Adjustment>, Invoice.RefundOptions).")]
```

Comment:
These are only available internally but useful as a reminder to remove them at some point.